content/sales.rst

```diff
@@ -0,0 +1,196 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describe operational Sales details.
+
+The main contact is Tim Adams, the main email group is at sales@proatria.com.
```

Comment:
in the styleguide we should not mention team members or email addressed

we should also not mention "SFTPPlus"

think of the styleguide as a generic project manual... and can be used for any project, not only for SFTPPlus

specific info should be for the wiki